kubexecutor/src/main/kotlin/pl/edu/prz/kod/application/ApplicationModule.kt

```diff
@@ -0,0 +1,11 @@
+package pl.edu.prz.kod.application
+
+import org.koin.dsl.module
+import pl.edu.prz.kod.domain.executor.ExecutorOrchestrator
+import pl.edu.prz.kod.ports.ExecutorOrchestratorPort
+import java.util.Base64
+
+val applicationModule = module {
+    single<Base64.Decoder> { Base64.getDecoder() }
```

Comment:
Seems like the generic type may be skipped here:

```suggestion
    single { Base64.getDecoder() }
```

However if you left it here for readibility reasons, that is ok. So treat this comment like something to consider rather than change request :)

Reply:
Sure, good point. I guess that for some reason intellij refused to cooperate when I tried to remove it :D